### PR TITLE
Onboarding Standalone Installation: Add new feature flag

### PR DESCRIPTION
**Pull request by jkguidaven**
#### Proposed Changes

* Add new `jetpack/standalone-plugin-onboarding-update-v1` feature flag for the new onboarding standalone installation.
* Add new copy to provide sample usage for the new feature flag.

#### Testing Instructions

1. * Run `git fetch && git checkout add/onboarding-standalone-installation-feature-flag`
    * Run `yarn start`
2. Go to http://calypso.localhost:3000/checkout/jetpack/thank-you/licensing-manual-activate-instructions/jetpack_backup_t1_yearly?flags=jetpack/standalone-plugin-onboarding-update-v1 or use the Calypso cloud live link and goto `/checkout/jetpack/thank-you/licensing-manual-activate-instructions/jetpack_backup_t1_yearly?flags=jetpack/standalone-plugin-onboarding-update-v1`.
3. Confirm the header displays **_Ok, let's install Jetpack Backup_.**
Screen Shot 2022-10-14 at 5 02 36 PM

#### Pre-merge Checklist

- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?

Related to 1203097885147382-as-1203159154260050

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203159154260048
  - https://app.asana.com/0/0/1203159154260050

### CODE REVIEW

**Comment by JessBoctor** [comment]
I did a functional review and it works as described: 

JB-CR-GH69063

Looks good to me.

File: config/development.json [hunk 1]
```diff
@@ -83,6 +83,7 @@
 		"jetpack/features-section/jetpack": true,
 		"jetpack/features-section/simple": true,
 		"jetpack/magic-link-signup": true,
+		"jetpack/onboarding-standalone-installation-v1": true,
```
Comment (manzoorwanijk) [inline config/development.json]: Let us name it something that makes the purpose clearer

```suggestion
		"jetpack/update-standalone-plugin-onboarding-v1": true,
```

Comment (manzoorwanijk) [reply config/development.json]: or

```suggestion
		"jetpack/standalone-plugin-onboarding-update-v1": true,
```

Comment (jkguidaven) [reply config/development.json]: I have renamed the feature flag to 'jetpack/standalone-plugin-onboarding-update-v1'  which describes the new feature more clearly.

**Review by manzoorwanijk — APPROVED** [review]
Looks great now. Thank you for applying the suggestions.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7581303

Thank you @jkguidaven for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7581303).

**Pull request merged**